src/main/java/GeneralService/Configurations.java

```diff
@@ -0,0 +1,46 @@
+package GeneralService;
+
+import onlineservices.OnlineService;
+import onlineservices.services.WindowControl.WindowControlService;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import utils.filereaders.CsvFileReader;
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class Configurations {
+    private static final Logger LOGGER = LogManager.getLogger(Configurations.class.getName());
+    @Autowired
+    private WindowControlService windowControlService;
+    @Bean
+    public WindowControlService windowControlService() {
+        return new WindowControlService();
+    }
+
+    public List<String> getActivatedServicesNames() {
+        CsvFileReader fileReader = new CsvFileReader();
+        return fileReader.readFile(new File(".\\src\\main\\resources\\ActivatedServices.csv"));
+    }
+
+    @Bean
+    public List<OnlineService> activatedServices() {
+        List<OnlineService> activatedServices = new ArrayList<>();
+        List<String> activatedServicesNames = getActivatedServicesNames();
+        for (String serviceName : activatedServicesNames) {
```

Comment:
can directly use getActivatedServicesNames(), no need to create a variable for it because you are not using that variable anywhere else